pkg/autoscaling/calculation.go

```diff
@@ -281,21 +282,37 @@ func getScaledGroupsByComponent(rc *cluster.RaftCluster, component ComponentType
 
 func getScaledTiKVGroups(informer core.StoreSetInformer, healthyInstances []instance) []*Plan {
 	planMap := make(map[string]map[string]struct{}, len(healthyInstances))
+	resourceTypeMap := make(map[string]string)
 	for _, instance := range healthyInstances {
 		store := informer.GetStore(instance.id)
 		if store == nil {
 			log.Warn("inconsistency between health instances and store status, exit auto-scaling calculation",
 				zap.Uint64("store-id", instance.id))
 			return nil
```

Comment:
We should throw the error here.

Reply:
throw what kind of error?

Reply:
I think the built-in error is enough currently. `getScaledTiKVGroups`  and `getScaledTiDBGroups` will return `[]*Plan` and `error` instead. we could handle the error later.

Reply:
Updated. How the API returns the error may be discussed later.